src/activity-log/index.html

```diff
@@ -17,13 +17,13 @@
     {% import 'macros/category-icon.html' as category_icon %}
     {% from 'post-macros.html' import pagination as pagination with context %}
 
-    <h1>Activity Log</h1>
-    <p class="h3">
+    <h1 data-qa-hook="title">Activity Log</h1>
```

Comment:
Cool, yeah just thinking that `title` sounds like the page's title, not the main content area title.